src/DurableTask.AzureStorage/Tracking/AzureTableTrackingStore.cs

```diff
@@ -846,6 +846,7 @@ public override Task StartAsync(CancellationToken cancellationToken = default)
                 bool isFinalEvent = i == newEvents.Count - 1;
 
                 HistoryEvent historyEvent = newEvents[i];
+                Utils.ConvertTimeToUtc(historyEvent);
```

Comment:
same as above, just a small nit for our own future reference:
```suggestion
                // For backwards compatibility, we convert timer timestamps to UTC prior to persisting to Azure Storage
                // see: https://github.com/Azure/durabletask/pull/1138
                Utils.ConvertTimeToUtc(historyEvent);
```